packages/connectkit/src/wallets/walletConfigs.tsx

```diff
@@ -337,9 +320,7 @@ export const walletConfigs: {
       website: "https://steakwallet.fi/download",
     },
     getWalletConnectDeeplink: (uri: string) => {
-      return isAndroid()
-        ? uri
-        : `https://links.steakwallet.fi/wc?uri=${encodeURIComponent(uri)}`;
+      return `https://links.steakwallet.fi/wc?uri=${encodeURIComponent(uri)}`;
```

Comment:
does this work on Android?

i think on Android they were linking to `wc:...` to invoke the Android "intent" = unlike on iOS, on Android multiple apps can register as handling a custom URI scheme